src/main/java/edu/hawaii/its/api/service/UpdateTimestampService.java

```diff
@@ -18,21 +22,24 @@
  */
 @Service("timestampService")
 public class UpdateTimestampService {
-
     public static final Log logger = LogFactory.getLog(MembershipService.class);
 
-    public GroupingTimestampResult update(GroupingResult groupingResult) {
+    @Autowired
+    public GroupPathService groupPathService;
+
+    public GroupingTimestampResults update(GroupingResult groupingResult) {
         if (groupingResult.getResultCode().equals("SUCCESS")) {
-            return updateLastModifiedTimestamp(groupingResult.getGroupPath());
+            List<String> groupPaths = new ArrayList<>();
+            groupPaths.addAll(groupPathService.getBasePathAndSubPathFromSubPath(groupingResult.getGroupPath()));
```

Comment:
These two lines can be simplified into one line: `List<String> groupPaths = groupPathService.getGroupingGroupPaths(groupingResult.getGroupPath())`